src/common/options.ts

```diff
@@ -26,6 +27,12 @@ export type DefaultTransactionOptions = DefaultOptions & {
   l2RpcUrl?: string;
   privateKey: string;
 };
+export type BalanceOptions = DefaultOptions & {
```

Comment:
I don't think we should keep `BalanceOptions` in this file. Let's move it to balance.ts where it is used.